### ISSUE TITLE
[Accessibility] - Sumo page text doesn't change its sizing 

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#24299]
**Steps to reproduce**

1. Increase text sizing from iOS settings.
1. Install FF and tap on the learn more link from the first onboarding card.
1. Observe the text size.

**Expected behavior**
The sumo page’s text size is scaled as per the iOS setting.

**Actual behavior**
The sumo page’s text size is NOT scaled as per the iOS setting.

**Device & build information**

* Device: iPad Air 11' (18.2)
* Build version: v135 (49962)
* First seen version: v132.1

Notes

Attachments:

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-11153)

**Comment by data-sync-user** [comment]
➤ Andres Furlan commented:

Diana Andreea Barladeanu does this work in other web views ? E.g ( http://E.gL ): The privacy notice. Thanks!

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Andres Furlan no, it doesn’t work.

**Comment by data-sync-user** [comment]
➤ Nishant Bhasin commented:

Diana Andreea Barladeanu I don’t know if this should be considered a bug because if you change A11Y text size in settings for the app, we change the app components but a website will still render in its own text size. 

Let me know if other browsers also don’t do this and we can look into it but otherwise we should close this. cc Andres Furlan thoughts?

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Nishant Bhasin Verified it on Chrome, it doesn’t adjust the sizing, either.

**Comment by data-sync-user** [comment]
➤ Andres Furlan commented:

Lucas Siebert can you report this to SUMO?
